docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,106 @@
+
+Manage Learners View
+--------------------
+
+The Manage Learners view is meant to be used by non-technical staff to administer enterprise learners.
+It can be used to:
+
+* Associate ("link") and disassociate ("unlink") existing students with Enterprise Customer.
+* Associate learners with the Enterprise customer even if they don't have an account yet (these
+  learners are known as "Pending Enterprise Customer Users"). Pending Enterprise Customer Users are
+  referenced/identified by their email address only.
+* Manually enroll groups of enterprise learners into courses and/or programs.
+
+.. image:: images/manage-learners.png
+
+Integrating with Course Catalog
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order for course enrollment features to work, ``edx-enterprise`` talks to the `Course Catalog Service`_. Instructions
+on setting up and configuring Course Catalog can be found at corresponding ``docs section`` of Course Catalog and in
+``edx-enterprise`` `Pull Request #7`_ (describes developer setup, but useful for a broader audience).
+
+One particular quirk is that Course Catalog Service, being an independent application, restricts access to
+courses/programs according to ownership rules, so only users with certain roles can list all programs and courses, as
+``edx-enterprise`` admin interface expects. This is covered in greater detail in aforementioned docs section and pull
+request.
+
+.. _Course Catalog Service: https://open-edx-course-catalog.readthedocs.io/en/latest/getting_started.html
+.. _docs section: https://open-edx-course-catalog.readthedocs.io/en/latest/getting_started.html#lms-integration
+.. _Pull Request #7: https://github.com/edx/edx-enterprise/pull/7
+
+Managing Linked Learners Lists
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Manage Learners view allows for two modes of linking users to an Enterprise Customer:
+
+* Singular - single user email or username is provided.
+* Bulk - CSV file containing single column of emails is uploaded.
+
+When linking a single user by username, ``edx-enterprise`` tries to find an existing user with that username and fails
+the linking if match was not found. When email is used (both in singular and bulk modes), existing users using that
+email are linked to Enterprise Customer; if email match was not found a "Pending linked learner" record is created for
+that email. If that email is used to register a new user that user is automatically linked with Enterprise customer.
+
+**Note:** at the moment of writing, each learner can be linked only to one Enterprise Customer, so linking fails if the
+learner is already associated with some other Enterprise Customer.
+
+The view also allows unlinking "Linked Learners" and "Pending Linked Learners".
+
+Enrolling Learners into Course or Program
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enrolling learners into course or program depends on Course Catalog integration. In two words, this feature uses
```

Comment:
Yeah, "In two words" is not an English idiom, so some people may find that strange :-p